find-start-lms.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
`depends_on` cant reach in to other docker compose files unless they are included by file... and if they are in the same directory... compose's executable confuses PWD/CWD contexting... 

So if im in `a`.... `
a$ docker compose -f ./docker-compose.yml -f ../b/docker-compose.yml up`... the docker compose config in `b` will be looking in a for all its includes, settings, dockerfiles, etc in `a`, instead of its own containing folder.

Reply:
Sure, but ill do so on Line 2 ;)